### PR TITLE
Card

### PR DESCRIPTION
**Pull request by reubenjnazareth**

### CODE REVIEW

File: card.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (yashpatel1797) [inline card.html:6]: you can add author tag

File: components/card.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    padding-bottom: 0.5rem;
+  }
+
+  .card-badge {
+      position:absolute;
+      right: 57rem;
+      top: 95rem;
+      background-color: #8d99ae;
+      padding: 0.5rem;
+      /* border-radius: 0.25rem; */
```
Comment (yashpatel1797) [inline components/card.css]: try to remove comments

File: components/card.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+  .dismiss-icon {
+    font-size: 2rem;
+    position: relative;
+    left: 14rem;
+    bottom: 22rem;
+}
+
+.dismiss-icon:hover {
+    color: #edf2f4;
```
Comment (yashpatel1797) [inline components/card.css:56]: you can use variable for color

**Review by yashpatel1797 — APPROVED** [review]
added a few suggestions look into it.

File: styles.css [hunk 4]
```diff
@@ -156,8 +156,20 @@ hr {
...
 
+.card-horizontal {
+    margin-left: 2rem;
+    margin-bottom: 1.25rem;
+}
+
+.card-vertical {
+    margin-left: 2rem;
+    margin-bottom: 1.25rem;
+}
```
Comment (adith-a-danthi) [inline styles.css:167]: Since both classes have the same properties, you can simplify it to a single class called `card`

**Review by adith-a-danthi — COMMENTED** [review]
Looks good mostly. A couple of points
- `card badge` seems to have some styling issues
- Try to use CSS variables for colors and shadows
- Format code before committing changes, formatting is not uniform

**Comment by reubenjnazareth** [comment]
yet to make corrections

**Pull request merged**